requirements.txt

```diff
@@ -7,8 +8,8 @@ arrow==0.12.1
 async-timeout==3.0.0
 binaryornot==0.4.4
 boto==2.48.0
-boto3==1.3.0
-botocore==1.4.93
+boto3>=1.3.0
+botocore>=1.4.93
```

Comment:
Typically, a requirements.txt is used for reproducible builds; you should pin every package to an exact version. setup.py (or requirements-minimal.txt - not sure why paasta has both) has loose minimum requirements.

I think paasta needs to decide if it's an application or a library, and then modify things accordingly. It might also be a bit of both, in which case things might be fine as-is (or it needs two separate requirements specifications). See https://github.com/Yelp/requirements-tools/blob/master/README.md for further information.

Reply:
It is kinda both :-( 

Generally we treat it as an application and discourage people from importing it. But there's at least 1 place I know we do end up importing it.

Reply:
@keymone I think @sjaensch is right in general though. Can we not just pin to a higher version or does that break another build?